docs/preview/gha-runner-scale-set-controller/README.md

```diff
@@ -138,6 +138,23 @@ $ kubectl logs -n "${NAMESPACE}" -l app.kubernetes.io/name=gha-runner-scale-set-
 kubectl logs -n "${NAMESPACE}" -l runner-scale-set-listener=arc-systems-arc-runner-set
 ```
 
+### Naming error: `Name must have up to characters`
+
+Because we generate labels and resource names based on the `AutoscalingRunnerSet`,
+the name of the installation must have up to 45 characters long. If you exceed the
+limit, the `helm install` command will fail with an error:
```

Comment:
We are using some of the resources generated names as labels for other resources. Resource names have a max length of `263 characters` while labels are limited to `63 characters`. Given this constraint, we have to limit the resource names to `63 characters`.

Since part of the resource name is defined by you, we have to impose a limit on the amount of characters you can use for the installation and namespace names.

If you see these errors, you have to use a shorter installation or namespace names.

Reply:
Thanks!